common/src/main/java/tachyon/exception/ExceptionMessage.java

```diff
@@ -99,7 +99,16 @@
   HDFS_FILE_NOT_FOUND("File {0} with id {1} is not found"),
 
   // file system master
+  FILE_MUST_HAVE_VALID_PARENT("{0} does not have a valid parent"),
```

Comment:
I don't think this would ever happen, but we do throw an exception if it occurs so I added it to the ExceptionMessages.